samples/toolbar/src/main/java/org/mozilla/samples/toolbar/SampleToolbarHelpers.kt

```diff
@@ -126,7 +126,7 @@ class UrlBoxProgressView(
         progressDrawable.setBounds(0, 0, w, h)
     }
 
-    override fun onDraw(canvas: Canvas?) {
+    override fun onDraw(canvas: Canvas) {
```

Comment:
I haven't found evidences that `Canvas` can be null when `onDraw()` is called. So I assume that it should be hadled as `@NonNull` and null-check here will be redundant.